### ISSUE TITLE
Guidance on Appropriate Use of Risk Scores in HCCEngine

### ISSUE THREAD

**Issue by eli-s-goldberg** [yubin-park/hccpy#72]
Hi Yubin!

Terrific work here. Top notch.  

## Description
We're using the `HCCEngine` from HCCPY in a causal ML model. The model is aimed at assessing the incremental impact on risk-adjusted revenue from our product. The model inputs include `Age`, `Sex`, `OREC`, `eligibility`, and an ESRD indicator. The `HCCEngine`'s profile function outputs four distinct risk scores: `score`, `score_age`, `score_adj`, and `score_age_adj`.

## Issue
Can you help us clarify the appropriate contexts for using each of these risk scores? Specifically, we need to understand if the choice of risk score depends on factors such as:

- The specific plan involved
- Individual member characteristics
- The nature or limitations of the data available

Any guidance or documentation regarding the conditional use of these scores would be greatly appreciated. Depending on the selection, we're seeing a significant dampening of risk factors to an unclear degree (nebulous, I know, but will provide some additional context, if needed). 

Thanks!

**Comment by ronnie-canopy** [comment]
What I summarize is better explained in this [pdf](https://www.hhs.gov/guidance/sites/default/files/hhs-guidance-documents/RiskAdjustmentMethodologyTranscript.pdf) direct from the CMS.

Risk scores produced when following the HCC methodology need to be adjusted via two factors:
- coding intensity factor
- normalization factor

These factors and explanations for them are available in the annual CMS Announcements.

Any risk score with <code>_adj</code> in the label refers to the score after normalization, otherwise the score is before normalization.
A risk score from the HCC model consists of two additive components:
- a demographic score based on age, gender, OREC, LTI, and Medicaid / Medicare dual status.
- a score based on HCC codes derived from input Dx codes (and some interactions of HCC codes)

The four scores you ask about are thus defined as:
| variable | description |
| :------- | :------------ |
| <code>score_age</code> | demo risk score only, before adjustments |
| <code>score_age_adj</code> | demo risk score only, after adjustments |
| <code>score</code> | sum of demo and Dx risk scores before adjustments |
| <code>score_adj</code> | sum of demo and Dx risk scores after adjustments |

This final row in the table above is what is used to calculate the CMS premium paid, and corresponds to the score on a CMS MMR report.

In conclusion, the need to use the factors does not depend on the bullets you listed.<br>
The scores before adjustments are useful for testing the model against CMS documentation.<br>
The scores after normalization are useful for the real world.

**Comment by eli-s-goldberg** [comment]
Terrific! We'll dive into this and about face with any questions. Standby.
